src/rdbms/mongoose_rdbms.erl

```diff
@@ -477,6 +479,12 @@ result_to_integer(Bin) when is_binary(Bin) ->
 selected_to_integer({selected, [{BInt}]}) ->
     result_to_integer(BInt).
 
+%% Converts a value from a CHAR field to integer
+%% PgSQL returns CHAR as an integer
+%% %% MySQL returns CHAR as a string
```

Comment:
Did you mean "binary"?

Reply:
I mean pgsql silently converts `CHAR(1)` into integer `UNSIGNED TINYINT` :D 
While mysql still returns it as a string data type `CHAR(size)`.

Reply:
What I mean is that the comments mention the types `integer` and `string` while the types accepted below are `binary` and `integer`. I think this comment is misleading as it suggests the Erlang type is `string()`, which is not.

Reply:
Changed this comment.